### DISCUSSION TITLE
App Crashing After Updating Play Core Library for First Deployment on Play Store

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#748] [Bugs]
Hi Team,

We are currently using release 5.0 and attempting to upload our app to the Play Store. This is the first deployment of the app on the platform.

During the deployment process, we encountered an error prompting us to update com.google.android.play:core from version 1.6.1 to 1.9.1. To resolve this, I updated the sb-cordova-plugin-utility using the following Git link: https://github.com/Sunbird-Ed/sb-cordova-plugin-utility.

However, after installing the APK, the app crashes. Below is a screenshot showing the error details.
![WhatsApp Image 2024-09-24 at 1 36 19 PM](https://github.com/user-attachments/assets/a45b4446-9086-4cb6-9503-6b4e37d20480)

Looking forward to your assistance.

cc: @sayali2523  @HarishGangula @swayangjit

**Comment by HarishGangula** [comment]
@siddhishinde0723 Can you share the crash logs if possible check the issue and suggest the changes.

**Reply by siddhishinde0723** [reply]
@HarishGangula  Here is the error log for the same 
![Screenshot_2024-09-24-15-51-03-632_com miui bugreport](https://github.com/user-attachments/assets/4dd33f12-1b69-43d7-8c5d-8e68eef638dc)
